apps/webview/src/app/[locale]/(home)/_components/CharacterView.tsx

```
@@ -0,0 +1,102 @@
+        foxRef.current.destroy();
+      }
+    };
+  }, [snowmanSvg, foxSvg]);
+
+  return (
+    <div className={containerStyle}>
+      <svg xmlns="http://www.w3.org/2000/svg" className={svgStyle} viewBox="0 0 375 199" fill="none">
+        {snowmanSvg && (
+          <g ref={snowmanElementRef} id="snowman-container" dangerouslySetInnerHTML={{ __html: snowmanSvg }}></g>
+        )}
+        {foxSvg && <g ref={foxElementRef} id="fox-container" dangerouslySetInnerHTML={{ __html: foxSvg }}></g>}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)